File: XM_Infor/XM_Infor/Classes/SelectCollectionViewCell.m
//
//  SelectCollectionViewCell.m
//  XM_Infor
//
//  Created by Robin He on 07/09/2017.
//  Copyright © 2017 Robin He. All rights reserved.
//

#import "SelectCollectionViewCell.h"

@interface SelectCollectionViewCell()


@end
@implementation SelectCollectionViewCell

//- (void)awakeFromNib {
//    [super awakeFromNib];
//    // Initialization code
//}

-(instancetype)initWithFrame:(CGRect)frame{
   
    if (self = [super initWithFrame:frame]) {
    
        self.selectImageV = [[UIImageView alloc]initWithFrame:CGRectMake(0, 100, 375, 300)];
        
        [self.contentView addSubview:self.selectImageV];
        
        self.selectImageV.clipsToBounds = YES;
        self.selectImageV.contentMode = UIViewContentModeScaleAspectFill;
    }
    
    return self;
}

@end

File: XM_Infor/XM_Infor/Classes/CreateExpense/VC/SelectCollectionViewCell.swift
//
//  SelectCollectionViewCell.swift
//  XM_Infor
//
//  Created by Robin He on 07/09/2017.
//  Copyright © 2017 Robin He. All rights reserved.
//

import UIKit

class SelectCollectionViewCell: UICollectionViewCell {
    
    var selectImageV:UIImageView?
    
    override init(frame: CGRect) {
        super.init(frame: frame)
        
        selectImageV = UIImageView()
//        selectImageV?.frame = CGRect(x: 0, y: 0, width: SCREEN_WIDTH, height: 400)
        
        contentView.addSubview(selectImageV!)
        
        
        
    }
    
    required init?(coder aDecoder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
    
}
